src/tests/test_kube_api.py

```diff
@@ -101,76 +423,29 @@ def unbound_highly_available_infraenv(
     ):
         infraenv_config, tf_config = kube_test_configs_late_binding_highly_available
         nodes = get_nodes_infraenv(tf_config, infraenv_config)
-        infra_env = kube_api_test_prepare_late_binding_infraenv(kube_api_context, nodes, infraenv_config)
+        api_client = kube_api_context.api_client
+        infra_env = self.kube_api_test_prepare_late_binding_infraenv(api_client, nodes, infraenv_config)
 
         yield infra_env, nodes
 
     @pytest.fixture
     @JunitFixtureTestCase()
     def unbound_single_node_cluster(self, kube_test_configs_single_node, kube_api_context):
         cluster_config, _ = kube_test_configs_single_node
-        yield kube_api_test_prepare_late_binding_cluster(
-            kube_api_context=kube_api_context, cluster_config=cluster_config, num_controlplane_agents=1
-        )
+        yield self.kube_api_test_prepare_late_binding_cluster(kube_api_context.api_client, cluster_config, 1)
```

Comment:
I think it's better to leave usage of keyword arguments on those kind places, even if it produces longer lines
No one can know what ``1`` means in this context without it, it's mainly a magic number

Reply:
here and elsewhere